css/style.css

```diff
@@ -1,19 +1,164 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
-body {
+body  {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    border: 2px solid red;
 }
 
+
 /**
  * Add your custom styles below
  * 
  * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
+ * - Be organized, use comments and separate your styles into meaningful chunks
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
 
+ 
+body{
+    width:100%;
+    display: flex;
+    flex-direction: column;
+    margin:0 auto;
+}
+/* body  *{
+    border:1px solid black;
+} */
+
+/* *******HEADER******* */
+.header{
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+} 
+.logo {
+    height: 80px;
+    width: 80px;
+    padding-left: 5rem;
+    margin: 2.5rem  0;
+}
+.nav_bar{
+    margin-top: 50px;
+}
+
+.navigation_list{
+    display:flex;
+    list-style: none;
+    padding-right: 5rem;  
+}
+
+.navigation_item{
+    padding:20px;
+    
+}
+
+.navigation_link {
+    color:rgb(109, 106, 106);
+    font-weight: 550;
+    font-size: 23px;
+    text-transform: uppercase;
+    text-decoration: none;
+
+}.navigation_link:hover{
+    color:rgb(245, 122, 21);
+}
+#meet_karma{
+    color:black;
+}
+#meet_karma:hover{
+    color:red;
+}
+
+/** MAIN***/
+
+.intro_karma{
+    width: 100%;
+    height: 100vh;
+    background-image: url("../img/first-background.jpg");  
+    background-size: cover; 
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    color:white;
+    margin-bottom: 50px;
+}
+
+.need_karma{
+    padding: 50px;
+}
+
+/*************TEXT STYLE*********/
+.intro_karma h1{
+    font-weight: 100;
+    font-size:70px;
+    margin-top:15rem;
+    
+}    
+.intro_karma h2{
+    font-weight: 50;
+    font-size: 40px;
+    margin-bottom:50px;
+}
+.need_karma h3{
+    text-align:center;
+    font-size: 50px;
+    font-weight: 40;
+    color: rgb(47, 45, 45);
+}
+.need_karma p{
+    color: rgb(47, 45, 45);
+    font-size:25px;
+
+}
+
+/********BUTTON********/
+button{
```

Comment:
Try to use class names instead of tag name, since buttons can be represented using button or input tags